projects/core/src/occ/adapters/organization/converters/occ-permission-type-list.normalizer.ts

```diff
@@ -0,0 +1,34 @@
+import { PERMISSION_TYPE_NORMALIZER } from 'projects/core/src/organization/connectors/permission-type';
+import { Injectable } from '@angular/core';
+import { Occ } from '../../../occ-models/occ.models';
+import {
+  Converter,
+  ConverterService,
+} from '../../../../util/converter.service';
+import { EntitiesModel } from '../../../../model/misc.model';
+import { OrderApprovalPermissionType } from 'projects/core/src/model/permission.model';
+
+@Injectable()
+export class OccPermissionTypeListNormalizer
+  implements
+    Converter<
+      Occ.OrderApprovalPermissionTypeList,
+      EntitiesModel<OrderApprovalPermissionType>
+    > {
+  constructor(private converter: ConverterService) {}
+
+  convert(
+    source: Occ.OrderApprovalPermissionType,
```

Comment:
I think here is a wrong type
```suggestion
    source: Occ.OrderApprovalPermissionTypeList,
```